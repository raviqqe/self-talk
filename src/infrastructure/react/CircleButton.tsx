import styled from "styled-components";
import { Button } from "./Button.js";

export const CircleButton = styled(Button)`
  font-size: 1.5em;
  padding: 0;
  height: 2em;
  width: 2em;
  border-radius: 1em;
`;
