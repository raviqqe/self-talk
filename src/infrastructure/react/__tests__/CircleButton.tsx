import { create } from "react-test-renderer";
import { CircleButton } from "../CircleButton";

it("renders", () => {
  expect(create(<CircleButton>foo</CircleButton>).toJSON()).toMatchSnapshot();
});

it("renders as a secondary button", () => {
  expect(
    create(<CircleButton secondary={true}>foo</CircleButton>).toJSON()
  ).toMatchSnapshot();
});
