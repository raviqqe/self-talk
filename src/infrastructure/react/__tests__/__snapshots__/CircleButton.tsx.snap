// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<button
  className="sc-AykKC sc-AykKD hfYJbS"
>
  foo
</button>
`;

exports[`renders as a secondary button 1`] = `
<button
  className="sc-AykKC sc-AykKD fyAhDl"
>
  foo
</button>
`;
